articles/client-platforms/angularjs/06-rules.md

```diff
@@ -1,128 +1,35 @@
 ---
 title: Rules
-description: test
+description: This tutorial will show you how to use Auth0 rules to extend what Auth0 has to offer.
 ---
 
 <%= include('../../_includes/_github', {
   link: 'https://github.com/auth0-samples/auth0-angularjs-sample/tree/master/06-Rules',
 }) %>_
 
-<!-- Every section in this doc can be reused in other quickstarts except the step 4 of creating rules  -->
+<%= include('../_includes/_rules-introduction.md') %>
```

Comment:
Should be `<%= include('../_includes/_rules-introduction') %>`

No `.md` on the path. I had to fix this in the other doc, rebase to see.